.github/workflows/process_new_pull_request.yml

```
@@ -0,0 +1,156 @@
+name: Assign Reviewer For Contributor
+
+on:
+  pull_request_target:
+    types: [opened]
+
+permissions: read-all
+
+jobs:
+  manage_reviewers:
+    name: Manage Reviewers and Notifications
+    runs-on: ubuntu-22.04
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Handle Reviewers and Notifications
+        uses: actions/github-script@v7
+        with:
+          script: |
+            // ======================
+            // INPUT SANITIZATION
+            // ======================
+            const safeGet = (obj, path, fallback = null) => {
+              return path.split('.').reduce((acc, key) => 
+                (acc && acc[key] !== undefined) ? acc[key] : fallback, obj
+              );
+            };
+
+            // Core PR metadata with validation
+            const prContext = context.payload.pull_request || {};
+            const repositoryContext = context.repo;
+            
+            // Validate repository context
+            const repoOwner = (repositoryContext.owner || '').replace(/[^a-zA-Z0-9-]/g, '');
+            const repoName = (repositoryContext.repo || '').replace(/[^a-zA-Z0-9-]/g, '');
```

I think your validations in #21705 were more comprehensive. Can you use those here too?

https://github.com/oppia/oppia/pull/21705/files#diff-fcd74b167efd4aa4c388702e0c5b923b17895432a82d50282f345acd2ec4806fR21

Also, if the repo owner has unexpected characters, we should abort instead of continuing after removing those characters. There's no good reason for those unexpected characters to be present, so if they are, it's dangerous to continue.

---

Done